fhir-audit/src/main/java/com/ibm/fhir/audit/logging/api/AuditLogServiceFactory.java

```diff
@@ -1,5 +1,5 @@
 /*
- * (C) Copyright IBM Corp. 2016,2019
+ * (C) Copyright IBM Corp. 2016,2020
```

Comment:
super-duper minor, but I usually put a space after the `,` as i edit it (a change from what we did when we auto-added these when first going open)

Reply:
✅